test/pnpm-lock.yaml

```diff
@@ -1,967 +1,637 @@
-lockfileVersion: '6.0'
+lockfileVersion: '9.0'
```

Comment:
- please avoid commiting generated file.
- an upgraded pnpm lock file will make fail all the CI.

please roll this file back to make the CI work.

or upgrate all workflow to make the CI back.

Reply:
if the CI pass, you are lucky.

Reply:
Reverted to previous version